src/tools/perf/api/libperf.h

```diff
@@ -124,25 +130,25 @@ enum {
      * 2) DPU offloading case: Sender DPU->Receiver DPU, (optional) AM message \
      *    to transfer data between DPU peers when AM communication is used. \
      *    For tag matching this message isn't used */ \
-    _macro(UCP_PERF_AM_ID_PEER_TX,              0) \
+    _macro(UCP_PERF_DAEMON_AM_ID_PEER_TX,       UCP_PERF_AM_ID) \
```

Comment:
maybe no need to define now and can always use `UCP_PERF_AM_ID`?

Reply:
IMO it's better to have it inside enum for consistency:
 - to uniformly treat all the IDs (including ID to string conversion) by ucp_perf_daemon_am_id_t
 - to enforce distinct values (it's indirectly enforced by switch in ucp_perf_daemon_am_id_name)